### PR TITLE
Adding new strings in Bengali

### PR DESCRIPTION
**Pull request by sffc**
Hi @Neilblaze,

I added some new strings in English in https://github.com/octave-online/oos-translations/commit/a350685c405a7dbcc2a23ac94e6dbfae18250ef2.  I used machine translation to generate Bengali for them.  Can you verify that the translations are ok?  Thank you!

### CODE REVIEW

File: locales/bn.json [hunk 1]
```diff
@@ -151,12 +159,12 @@
...
-  "header.buckets.delete#btn": {
-    "message": "ডিলিট বাকেট",
+  "header.buckets.delete#btn@2": {
+    "message": "ডিলিট বাকেট/প্রকল্প",
     "description": "Button text"
   },
-  "header.buckets.title": {
-    "message": "আপনার বাকেট:",
+  "header.buckets.title@2": {
+    "message": "বালতি এবং প্রকল্প:",
```
Comment (Neilblaze) [inline locales/bn.json]: replace via this :arrow_right: `"message": "আপনার বাকেট এবং প্রকল্প:"`

File: locales/bn.json [hunk 2]
```diff
@@ -419,6 +447,10 @@
     "message": "বাকেটের মধ্যে ফাইলস",
     "description": "Section header, followed by a list"
   },
+  "modals.createbucket.subhead3p": {
+    "message": "প্রকল্পে মধ্যে ফাইলস",
```
Comment (Neilblaze) [inline locales/bn.json]: Replace via this :arrow_right: `"message": "প্রকল্পের মধ্যে ফাইলস"`

**Review by Neilblaze — CHANGES_REQUESTED** [review]
Two minor changes, & you're good to go! :tada: 
Never seen so perfect translation. :laughing:

**Comment by sffc** [comment]
Thanks!

**Pull request merged**